packages/fileeditor-extension/src/index.ts

```diff
@@ -64,7 +60,7 @@ const FACTORY = 'Editor';
 /**
  * The command IDs used by the fileeditor plugin.
  */
-namespace CommandIDs {
+export namespace CommandIDs {
```

Comment:
Let's put these in the commands file or in a separate file so we don't have circular imports (commands imports index, index imports commands)

Reply:
I can move these in this PR, I had made that change in my one of follow up branch anyway. (I have a set of branches ready with the follow up steps after this PR)